### PR TITLE
Manuscript

### PR DESCRIPTION
**Pull request by stephendl102**

### CODE REVIEW

**Comment by stephendl102** [comment]
@aarmey  I noticed the build failed, and I believe its because it timed out after 6 hours. I knew the fitting would make it take longer, but I didn't know it would take that long. Is there something we can do about this? Or is there an error somewhere that you could point me to?

**Comment by aarmey** [comment]
What specifically is changing here? I see a ton of changes, so I'm not really sure what to look at.

**Comment by stephendl102** [comment]
@aarmey There are a few differences. The first is that I edited the manuscript. Second, I added a figure 4 to build for the manuscript. Third, I provided an upper and lower bound for fit activity so that fitRegression() actually fits the activity rather than just using the constant vector; this allows the synergy plots to be formed based off of fit activities depending on the disease model. Lastly, I allowed optional arguments for many of the functions to allow for specification of single receptors, single cell type, disease type, receptor binding/induced activity/depletion. I just reduced the upper and lower bounds from five to four in the hopes that this would allow the figures to build faster.

**Comment by aarmey** [comment]
These should be separate pull requests because they do different things. This will also help to uncover what the problem is.

File: src/figures/figureW.jl [hunk 1]
```diff
@@ -126,18 +129,27 @@ function L0fSearchHeatmap(df, dataType, vmax, clmin, clmax; murine = true)
...
+    elseif Cellidx != nothing # look at only one cell FcExpr
         FcExpr = importRtot(murine = murine)[:, Cellidx]
+        ylabel = "Activity"
+    else
+        FcExpr = importRtot(; murine = murine)
+    end
+    if fit == nothing
+        title = "Not fit"
+    else
+        title = "dataType"
```
Comment (cyrillustan) [inline src/figures/figureW.jl:149]: @stephendl102 I'm not sure what you wanted to do here. You want the word "not fit" and "dataType" in the parenthesis on the plot?

Comment (stephendl102) [reply src/figures/figureW.jl:149]: @cyrillustan As it stands currently, synergy can be calculated from a mixture with a guess of activities vector ([-1,0,1,1]), or a disease model is provided and the activities are fit during regression. I wanted to be able to distinguish this on the plot itself so as not to confuse when regressed activities are used or not.

**Pull request merged**